.teamcityTest/Gradle_Check_Tests/CIConfigIntegrationTests.kt

```diff
@@ -234,6 +232,15 @@ class CIConfigIntegrationTests {
         }
     }
 
+    @Test
+    fun uuidsAreUnique() {
+        val uuidList = CIBuildModel().stages.flatMap { it.functionalTests.map { ft -> ft.uuid  } }
+        val uuidSet = HashSet<Int>()
+        uuidList.forEach {
+            assertTrue(uuidSet.add(it), "Functional test UUID is not unique: $it")
```

Comment:
You could do 
```
assertEquals(uuidList.distinct(), uuidList)
```

Reply:
I wanted to avoid it to display which uuid is duplicated in the assertion failure message.

Reply:
It is quite easy to see in IDEA (show diff). Though on CI I think you are right that the duplicate ID may be harder to find.